controller/src/main/java/gov/va/vro/controller/VroController.java

```diff
@@ -95,6 +95,9 @@ public ResponseEntity<Object> fetchPdf(String claimSubmissionId)
 
       log.info("RESPONSE from fetchPdf: {}", pdfResponse.toString());
       if (pdfResponse.hasContent()) {
+        if (pdfResponse.getStatus() == "NOT_FOUND") {
+          return new ResponseEntity<>(pdfResponse, HttpStatus.NOT_FOUND);
+        }
```

Comment:
We are now handling NOT_FOUND but not ERROR. Error should be handled too.

Reply:
The catch(Exception) block should handle all other errors. They are redirected to GlobalExceptionHandler

Reply:
Right but it is not an exception. We return status ERROR from the Python code (exception is handled in the python code).